### ISSUE TITLE
Invalid indentation isn't reported as an error

### ISSUE THREAD

**Issue by jsdevel** [prettier/stylelint-config-prettier#24]
I have the following config:

```
{
  "extends": ["stylelint-config-prettier"]
}
```

**Comment by shannonmoeller** [comment]
This is intended behavior as prettier (which is assumed to be running separately) handles indentation.

**Comment by jsdevel** [comment]
Is there a way to force it?  I find it hard to justify the intention when other settings are handled by this config.

**Comment by shannonmoeller** [comment]
The whole point of `stylelint-config-prettier` is to turn off rules that `prettier` handles for you. This config doesn't run `prettier` on your code, it just removes `stylelint` rules that conflict with running `prettier`.

**Comment by shannonmoeller** [comment]
Hi, @jsdev! Just following up to see if you were able to get stylelint and prettier working together well. Happy to help if you’re still having issues.
